util/util.go

```diff
@@ -380,3 +380,13 @@ func HasOwner(refList []metav1.OwnerReference, apiVersion string, kinds []string
 
 	return false
 }
+
+// IsPaused returns true if the object has the `pause-reconciler` annotation.
+func IsPaused(v metav1.Object) bool {
```

Comment:
`IsPaused` might lead to confusion on resources that have a `Paused` field (such as `MachineDeployments`).